src/ImageSharp/Primitives/Matrix5x4.cs

```diff
@@ -0,0 +1,459 @@
+            m.M52 = value1.M52 + value2.M52;
+            m.M53 = value1.M53 + value2.M53;
+            m.M54 = value1.M54 + value2.M54;
+
+            return m;
+        }
+
+        /// <summary>
+        /// Subtracts the second matrix from the first.
+        /// </summary>
+        /// <param name="value1">The first source matrix.</param>
+        /// <param name="value2">The second source matrix.</param>
+        /// <returns>The result of the subtraction.</returns>
+        public static Matrix5x4 operator -(Matrix5x4 value1, Matrix5x4 value2)
+        {
+            Matrix5x4 m;
+
+            m.M11 = value1.M11 - value2.M11;
+            m.M12 = value1.M12 - value2.M12;
+            m.M13 = value1.M13 - value2.M13;
+            m.M14 = value1.M14 - value2.M14;
+            m.M21 = value1.M21 - value2.M21;
+            m.M22 = value1.M22 - value2.M22;
+            m.M23 = value1.M23 - value2.M23;
+            m.M24 = value1.M24 - value2.M24;
+            m.M31 = value1.M31 - value2.M31;
+            m.M32 = value1.M32 - value2.M32;
+            m.M33 = value1.M33 - value2.M33;
+            m.M34 = value1.M34 - value2.M34;
+            m.M41 = value1.M41 - value2.M41;
+            m.M42 = value1.M42 - value2.M42;
+            m.M43 = value1.M43 - value2.M43;
+            m.M44 = value1.M44 - value2.M44;
+            m.M51 = value1.M51 - value2.M51;
+            m.M52 = value1.M52 - value2.M52;
+            m.M53 = value1.M53 - value2.M53;
+            m.M54 = value1.M54 - value2.M54;
+
+            return m;
+        }
+
+        /// <summary>
+        /// Returns a new matrix with the negated elements of the given matrix.
+        /// </summary>
+        /// <param name="value">The source matrix.</param>
+        /// <returns>The negated matrix.</returns>
+        public static unsafe Matrix5x4 operator -(Matrix5x4 value)
+        {
+            Matrix5x4 m;
+
+            m.M11 = -value.M11;
+            m.M12 = -value.M12;
+            m.M13 = -value.M13;
+            m.M14 = -value.M14;
+            m.M21 = -value.M21;
+            m.M22 = -value.M22;
+            m.M23 = -value.M23;
+            m.M24 = -value.M24;
+            m.M31 = -value.M31;
+            m.M32 = -value.M32;
+            m.M33 = -value.M33;
+            m.M34 = -value.M34;
+            m.M41 = -value.M41;
+            m.M42 = -value.M42;
+            m.M43 = -value.M43;
+            m.M44 = -value.M44;
+            m.M51 = -value.M51;
+            m.M52 = -value.M52;
+            m.M53 = -value.M53;
+            m.M54 = -value.M54;
+
+            return m;
+        }
+
+        /// <summary>
+        /// Multiplies a matrix by another matrix.
+        /// </summary>
+        /// <param name="value1">The first source matrix.</param>
+        /// <param name="value2">The second source matrix.</param>
+        /// <returns>The result of the multiplication.</returns>
+        public static Matrix5x4 operator *(Matrix5x4 value1, Matrix5x4 value2)
```

Comment:
The result of this multiplication should be [either a `5x5` or a `4x4` matrix](https://en.wikipedia.org/wiki/Matrix_multiplication), depending on how we define the operation.

As for non-square fixed size matrices the multiplication has usually no practical use, I suggest to drop this operation.

Reply:
As you touched upon below this does make sense if our struct is an optimized form of a `Matrix5x5` in the manner that `Matrix3x2` is an optimized `Matrix3x3` which it is. 

Most importantly it works!
https://github.com/SixLabors/ImageSharp/blob/f10a96c2d9b288019e7a4b3bcca04cd7ce87179f/tests/ImageSharp.Tests/Processing/Processors/Filters/FilterTest.cs#L45